maven-plugin/pom.xml

```diff
@@ -56,5 +61,78 @@
       <artifactId>access-modifier-annotation</artifactId>
       <version>1.4</version>
     </dependency>
+    <dependency>
```

Comment:
for Integration testing -
https://khmarbaise.github.io/maven-it-extension/itf-documentation/usersguide/usersguide.html
https://dzone.com/articles/maven-plugin-testing-in-a-modern-way-part-i